src/test/java/integration/EventsTest.java

```diff
@@ -882,8 +884,9 @@ public void given_multibranchWithSourcesAndTwoBranches_when_indexing_then_deadBr
             prj.scheduleBuild2(0).getFuture().get();
             r.waitUntilNoActivity();
             assertThat("Indexing applies dead branch cleanup",
-                    prj.getItems(), containsInAnyOrder(master));
```

Comment:
@stephenc sorry but could you point me to the place where that is being checked? Just so that I am aware.